components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -1460,6 +1460,31 @@ public class MonitoredItemWrapper {
                         _pendingAlarmsSnapshotTimer.Start();
                         return;
                     }
+                    else if (eventType == ObjectTypeIds.RefreshRequiredEventType) {
+                        var noErrorFound = true;
+
+                        // issue a condition refresh to make sure we are in a correct state
+                        _logger.Information("Now issuing ConditionRefresh for subscription " +
```

Comment:
has there been an ask to support conditionrefresh2 as well?

Reply:
No, not that I have heard about.